crates/orchestrator/src/lib.rs

```diff
@@ -246,70 +252,11 @@ pub trait OrchestratorApi<KEY: SignatureKey> {
     fn get_builders(&self) -> Result<Vec<Url>, ServerError>;
 }
 
-impl<KEY> OrchestratorApi<KEY> for OrchestratorState<KEY>
+impl<KEY> OrchestratorState<KEY>
 where
     KEY: serde::Serialize + Clone + SignatureKey + 'static,
 {
-    /// Post an identity to the orchestrator. Takes in optional
-    /// arguments so others can identify us on the Libp2p network.
-    /// # Errors
-    /// If we were unable to serve the request
-    fn post_identity(
-        &mut self,
-        libp2p_address: Option<Multiaddr>,
-        libp2p_public_key: Option<PeerId>,
-    ) -> Result<u16, ServerError> {
-        let node_index = self.latest_index;
-        self.latest_index += 1;
-
-        if usize::from(node_index) >= self.config.config.num_nodes_with_stake.get() {
-            return Err(ServerError {
-                status: tide_disco::StatusCode::BAD_REQUEST,
-                message: "Network has reached capacity".to_string(),
-            });
-        }
-
-        // If the orchestrator is set up for libp2p and we have supplied the proper
-        // Libp2p data, add our node to the list of bootstrap nodes.
-        if self.config.libp2p_config.clone().is_some() {
-            if let (Some(libp2p_public_key), Some(libp2p_address)) =
-                (libp2p_public_key, libp2p_address)
-            {
-                // Push to our bootstrap nodes
-                self.config
-                    .libp2p_config
-                    .as_mut()
-                    .unwrap()
-                    .bootstrap_nodes
-                    .push((libp2p_public_key, libp2p_address));
-            }
-        }
-        Ok(node_index)
-    }
-
-    // Assumes nodes will set their own index that they received from the
-    // 'identity' endpoint
-    fn post_getconfig(&mut self, _node_index: u16) -> Result<NetworkConfig<KEY>, ServerError> {
-        Ok(self.config.clone())
-    }
-
-    // Assumes one node do not get twice
-    fn get_tmp_node_index(&mut self) -> Result<u16, ServerError> {
-        let tmp_node_index = self.tmp_latest_index;
-        self.tmp_latest_index += 1;
-
-        if usize::from(tmp_node_index) >= self.config.config.num_nodes_with_stake.get() {
-            return Err(ServerError {
-                status: tide_disco::StatusCode::BAD_REQUEST,
-                message: "Node index getter for key pair generation has reached capacity"
-                    .to_string(),
-            });
-        }
-        Ok(tmp_node_index)
-    }
-
-    #[allow(clippy::cast_possible_truncation)]
-    fn register_public_key(
+    fn register_unknown(
```

Comment:
Any reason for the name change here?

Reply:
left a longer comment in another response that should explain this!